Packs/CommonScripts/Scripts/CommandLineAnalysis/CommandLineAnalysis.py

```diff
@@ -92,7 +92,7 @@
 
 LATERAL_MOVEMENT = [
     r"\\\\[a-zA-Z0-9_.-]+\\C\$\b",
-    r"\b(?:cmd(?:\.exe)?)\s+(?=.*\/q)(?=.*\/c).*?((?:1>\s?.*?)?\s*2>&1)\b",
+    r"(?i)\b(?P<filename>cmd[.]exe)[\"']?(?P<flags>(?:\s+/[qc])+)(?P<args>.+?[\S])\s+(?:1>>?\s*)(?P<export_path>.+?)\s*?(?:2>&1)\b",
```

Comment:
QQ:
I see this regex won't catch 
`"C:\Windows\System32\cmd.exe" /q /c ipconfig >>aaa.txt`
Is that on purpose, or is there another regex that catches double `>>`

Reply:
Ive removed it and swapped it to 2 different patterns:
1. `(?i)\b(?P<filename>cmd[.]exe)[\"']?(?P<flags>(?:\s+/[qc])+)\b`
2. `(?i)\b(?:2>&1)\b`
The 2 new patterns are under `SUSPICIOUS_COMMAND_PATTERNS` as it is a better fit than `LATERAL MOVEMENT`